command/server/config.go

```diff
@@ -77,6 +78,9 @@ type Config struct {
 	EnableResponseHeaderHostname    bool        `hcl:"-"`
 	EnableResponseHeaderHostnameRaw interface{} `hcl:"enable_response_header_hostname"`
 
+	LogRequestsInfo    log.Level   `hcl:"-"`
```

Comment:
It feels like making this field log.Level might be more trouble than it's worth.  I was just thinking that the Core field (that's used all the time) should be log.Level.

Reply:
Also, maybe the name should be LogRequestsLevel (and the HCL field `log_requests_level`)?

Reply:
Since the Core field would need protection for synchronization issues, how about I keep that as *uberatomic.Int32.

Reply:
Yeah, sorry, that's what I meant.